Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -1969,10 +1975,15 @@ void WebDriverConnection::restore_the_window()
     m_page_client->page_did_request_restore_window();
 
     // Do not return from this operation until the visibility state of the top-level browsing context’s active document has reached the visible state, or until the operation times out.
-    // FIXME: Implement timeouts.
-    Web::Platform::EventLoopPlugin::the().spin_until([this]() {
+    // FIXME: It isn't clear which timeout should be used here.
```

Comment:
It is not clear what is meant by "[...] until the operation times out" here. I've used the page load timeout, but I'm not sure it is supposed to be used in this case.